flink-libraries/flink-table/src/main/scala/org/apache/flink/table/sources/FilterableTableSource.scala

```diff
@@ -55,4 +55,9 @@ trait FilterableTableSource[T] {
     */
   def isFilterPushedDown: Boolean
 
+  /**
+    * @param relBuilder Builder for relational expressions.
+    */
+  def setRelBuilder(relBuilder: RelBuilder): Unit
```

Comment:
Can you move this method to PartitionableTableSource?

Reply:
`setRelBuilder` method is called in `PushFilterIntoTableSourceScanRule`. If we move  `setRelBuilder` method to `PartitionableTableSource`,  `PushFilterIntoTableSourceScanRule` should know `FilterableTableSource` and `PartitionableTableSource` both.